src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -378,7 +378,11 @@ public static Config BaseConfig
         private static string GetCallerName()
         {
             var @this = typeof(MultiNodeSpec).Name;
+#if CORECLR
+            var trace = new StackTrace(new Exception(), false);
```

Comment:
Is this works find on `NetCore`? StackTrace is using to get the proper name for test suite, not for exceptions